src/Trixi.jl

```diff
@@ -200,7 +202,8 @@ export source_terms_harmonic
 export initial_condition_poisson_nonperiodic, source_terms_poisson_nonperiodic,
        boundary_condition_poisson_nonperiodic
 export initial_condition_eoc_test_coupled_euler_gravity,
-       source_terms_eoc_test_coupled_euler_gravity, source_terms_eoc_test_euler
+       source_terms_eoc_test_coupled_euler_gravity, source_terms_eoc_test_euler,
+       source_terms_eoc_test_polytropic
```

Comment:
```suggestion
       source_terms_eoc_test_coupled_euler_gravity, source_terms_eoc_test_euler
```
AFAICT, this function was renamed and does not need an export anymore/